src/components/molecules/pager/pager.jsx

```diff
@@ -66,21 +67,22 @@ const Pager = ({ current = 1, currentPageNeighbours = 2, nextLabel = 'Next >', o
 
 			if (isNumberVisible) {
 				pagerItem = (
-					<li className="pager__list-item" key={`page-${pageNumber}`}>
-						<button className={`pager__button${pageNumber === currentPage ? ' active' : ''}`} onClick={() => pageNavigationChangeHandler(pageNumber)} aria-label={`page ${pageNumber}`}>
+					<li className="usa-pagination__item usa-pagination__page-no" key={`page-${pageNumber}`}>
+						{/* eslint-disable-next-line */}
+						<a href="#" className={`usa-pagination__button${pageNumber === currentPage ? ' usa-current' : ''}`} onClick={() => pageNavigationChangeHandler(pageNumber)} aria-label={`Page ${pageNumber}`}>
```

Comment:
I believe this is appended to the end of the URL when the next or previous button which is a bit odd

Reply:
Discussed in person---URL is fine on this branch